apps/ledger-live-desktop/src/renderer/components/DeviceAction/index.tsx

```diff
@@ -318,6 +321,15 @@ export const DeviceActionDefaultRendering = <R, H extends States, P>({
     isTrackingEnabled: useSelector(trackingEnabledSelector),
   });
 
+  useTrackTransactionChecksFlow({
+    location,
+    device,
+    deviceInfo,
+    appAndVersion,
+    transactionChecksOptInTriggered,
+    isTrackingEnabled: useSelector(trackingEnabledSelector),
```

Comment:
Could we move the `useSelector` in a const and only call the useSelector once please ?